semantic-web-technology-introduction.markdown

```diff
@@ -12,26 +12,25 @@ In general a growing number of people produce in their activities, both professi
 The SW standards enhance the interoperability of computer systems through the introduction of **machine interpretable or formal semantics** of natural language.  
-For clarity it has to be said that the novel element in the SW is not the semantics, but the Web. First there was natural language and its semantics for  milennia, and later on logics of all kinds. Then there was Information Technology, the Internet connecting computers, and the Web connecting pages, documents and applications. And finally the SW environment with its standards is able to formalize the semantics of natural language and make it machine interpretable, which enables linking information resources in a meaningful way.
+For clarity it has to be said that the novel element in the SW is not the semantics, but the Web. The development towards this Web can be explained shortly like this: First there was natural language and its semantics and later on logics of all kinds. Then there was Information Technology, the Internet connecting computers, and the Web connecting pages, documents and applications. And finally the SW environment with its standards is able to formalize the semantics of natural language and make it machine interpretable, which enables linking information resources in a meaningful way.
 
 Figure 1 shows the common representation of the different technologies with their respective standards comprised in the SW stack.   
 
 {% include image.html type="small-figure" url="/assets/images/semantic-web-stack.png" description="Figure 1: The Semantic Web Stack" %}
 
 ### Identify: IRI
-The Web already comes with digital resources bearing an IRI or Internationalized Resource Identifier (URI or Uniform Resource Identifier and URL or Uniform Resource Locator are respective sub-concepts). So, it is already good to know what resource an agent—person or machine—is speaking about because the former is identified, and if another agent looks for it, it can find it (if accessible).  
-The next step is for the other agent—again person or machine—to understand what the resource is about. The fact that also the machine 'understands' content is the game changer. Therefore it needs explicit formal semantics based on logic. More precisely the different standard languages (except SPARQL) have their [model (or interpretation) theory](https://en.wikipedia.org/wiki/Model_theory) based on [first order logic](https://en.wikipedia.org/wiki/First-order_logic) and [set theory](https://en.wikipedia.org/wiki/Set_theory).
+The Web already comes with digital resources bearing an IRI or Internationalized Resource Identifier (URI or Uniform Resource Identifier and URL or Uniform Resource Locator are respective sub-concepts). # So, it is already good to know what resource an agent — person or machine — is speaking about because the former is identified, and if another agent looks for it, it can find it (if accessible). #
```

Comment:
_# So, it is already good to know what resource an agent — person or machine — is speaking about because the former is identified, and if another agent looks for it, it can find it (if accessible). #_: I would rewrite this sentence (or maybe even omit). It is a bit circumstantially expressed. May be: _So beeing able to determine via identifiers what an agent - person or machine - describes is already a first step._ And maybe the next sentence like this: _The second step for the other agents (person or machine) is, to understand what the resource is about._

Reply:
OK